lib/ansible_tower_client/base_models/project.rb

```diff
@@ -19,8 +19,19 @@ def update
     end
 
     def last_update
-      return if related['last_update'].blank?
-      api.project_updates.find(related['last_update'])
+      return @last_update if defined? @last_update
+      return @last_update = nil unless related.raw_hash.key?('last_update')
```

Comment:
If the key is not there now, it won't be populated later?

Reply:
No, if it was once nil, it will be nil further. That's why I check with `defined?`. This way I'm preventing multiple API requests.